game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleState.java

```diff
@@ -102,6 +102,8 @@ public boolean isFirstRound() {
 
   Collection<Unit> getDependentUnits(Collection<Unit> units);
 
+  void removeDependentUnits(Collection<Unit> units);
```

Comment:
There's now 3 methods dealing with dependents (`getDependentUnits`, `removeDependentUnits`, and `getTransportDependents`).  I'm looking into if I can simplify the api around dependent units.  I expect `getTransportDependents` and `getDependentUnits` can be combined somehow.  I also think that they might be combined with `filterUnits` by adding a new status `TRANSPORTING`.  But that will be a different PR once.